### PR TITLE
REL scikit-learn 1.3.2

### PR DESCRIPTION
**Pull request by glemaitre**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by glemaitre** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -114,9 +114,11 @@ test:
     # reduce contention for parallelism (xdist-workers, openblas, openmp) that
     # actually increases the runtime and can lead to timeouts
     - export OMP_NUM_THREADS=1                 # [aarch64 or ppc64le]
-    - pytest --pyargs sklearn -k "not ({{ tests_to_skip }})" -nauto --timeout=1200 --durations=50  # [not ppc64le]
+    - pytest --pyargs sklearn -k "not ({{ tests_to_skip }})" -nauto --timeout=1200 --durations=50  # [not ppc64le or not aarch64]
```
Comment (ogrisel) [inline recipe/meta.yaml:117]: Apparently this does not work as intended: the tests are still run on the last commit.

**Comment by glemaitre** [comment]
@conda-forge-admin, please rerender

**Comment by glemaitre** [comment]
I am merging now but we should figure out the weird behaviour for `aarch64`.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Should it be?

```suggestion
    - pytest --pyargs sklearn -k "not ({{ tests_to_skip }})" -nauto --timeout=1200 --durations=50  # [not (ppc64le or aarch64)]
```

**Comment by jakirkham** [comment]
Ok it would be nice to have a bit more detail about the ARM test failures in a new feedstock issue

**Pull request merged**